### PR TITLE
komb

### PR DESCRIPTION
**Pull request by advaitb**
Creating a recipe for a new tool ->  KOMB: Characterizing metagenomes using K-core decomposition.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by advaitb** [comment]
@BiocondaBot please add label

**Comment by advaitb** [comment]
@BiocondaBot please fetch artifacts

File: recipes/komb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - bifrost >=1.0.5
+    - igraph >=0.7.1
+    - zlib
+test:
+  commands:
+    - komb -h
+
+about:
+  home: https://gitlab.com/treangenlab/komb
+  license: GPLv3
```
Comment (bgruening) [inline recipes/komb/meta.yaml:39]: can you pleae use spdx idetifiers

File: recipes/komb/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package:
+  name: komb
+  version: "1.0"
+
+source:
+  url: https://gitlab.com/treangenlab/komb/-/archive/v1.0/komb-v1.0.tar.gz
```
Comment (bgruening) [inline recipes/komb/meta.yaml:6]: This link looks wired. Can you use the classical one and replace the version number with a templated version?

**Review by bgruening — COMMENTED** [review]
Thanks @advaitb. Two small comments.

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Comment by advaitb** [comment]
@dpryan79 Thanks for the edits and the merge. Does the python requirement in the meta.yaml default to 2.7 ? I had it as >= 3.5 earlier and  the current  package when installed from conda seems to complain about Python 3.x

**Pull request merged**